src/views/SearchView.vue

```diff
@@ -51,10 +55,24 @@ export default class SearchView extends Vue {
     }
 
     async onSearch(searchData: SearchData<void>) {
+        try {
```

Comment:
MGViereck I think it makes sense to set ```this.error=""``` when search is submitted to hide the last error message.

Reply:
is adapted in commit comment 1